### PR TITLE
btm, bup: add pt_BR translation

### PR DESCRIPTION
**Pull request by gabxyz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/btm.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+`btm --dot_marker`
+
+- Exibe também a carga da bateria e o estado de saúde:
+
+`btm --battery`
+
+- Atualiza a cada 250 milissegundos e exibe os últimos 30 segundos nos gráficos:
+
+`btm --rate 250 --default_time_value 30000`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/btm.md:25]: The following code fix can be applied to the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/btm.md) page and then here:

```suggestion
`btm --rate {{250}} --default_time_value {{30000}}`
```

File: pages.pt_BR/common/bup.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# bup
+
+> Sistema de backup baseado no formato Git packfile, oferecendo salvamentos incrementais e desduplicação global.
+> Mais informações: <https://github.com/bup/bup>.
+
+- Inicializa um repositório de backup no diretório local especificado:
+
+`bup -d {{caminho/para/repositório}} init`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/bup.md:8]: The following example fix can be applied to the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/bup.md) page and then here:

```suggestion
- Inicializa um repositório de backup no [d]iretório local especificado:

`bup -d {{caminho/para/repositório}} init`
```

File: pages.pt_BR/common/bup.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Prepara um determinado diretório antes de fazer um backup:
+
+`bup -d {{caminho/para/repositório}} index {{caminho/para/diretório}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/bup.md:12]: The following example fix can be applied to the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/bup.md) page and then here:

```suggestion
- Prepara um determinado [d]iretório antes de fazer um backup:

`bup -d {{caminho/para/repositório}} index {{caminho/para/diretório}}`
```

File: pages.pt_BR/common/bup.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Faz o backup de um diretório para o repositório:
+
+`bup -d {{caminho/para/repositório}} save -n {{nome_do_backup}} {{caminho/para/diretório}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/bup.md:16]: The following example fix can be applied to the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/bup.md) page and then here:

```suggestion
- Faz o backup de um [d]iretório para o repositório:

`bup -d {{caminho/para/repositório}} save -n {{nome_do_backup}} {{caminho/para/diretório}}`
```

File: pages.pt_BR/common/bup.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Exibe os snapshots de backup armazenados atualmente no repositório:
+
+`bup -d {{caminho/para/repositório}} ls`
+
+- Restaura um snapshot de backup específico para um diretório de destino:
+
+`bup -d {{caminho/para/repositório}} restore -C {{caminho/para/diretório_de_destino}} {{nome_do_backup}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/bup.md:24]: The following example fix can be applied to the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/bup.md) page and then here:

```suggestion
- Restaura um snapshot de backup específico para um [d]iretório de destino:

`bup -d {{caminho/para/repositório}} restore -C {{caminho/para/diretório_de_destino}} {{nome_do_backup}}`
```

**Review by EmilySeville7cfg — APPROVED** [review]
- No changes are requested because translations seem to be synced with the original pages even the last ones are required to be changed.
- More accurate grammar checks should be done by other reviewers.

File: pages.pt_BR/common/btm.md (hunk 1 above)
Comment (navarroaxel) [reply pages.pt_BR/common/btm.md:25]: Not in this case, because it isn't a placeholder. It's a specific value in the example

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**